automation/common.sh

```diff
@@ -3,6 +3,16 @@
 # Common functions for the scripts
 #
 
+
+set_guestfs_params() {
+    # see: https://bugzilla.redhat.com/show_bug.cgi?id=1404287
+    export LIBGUESTFS_APPEND="edd=off"
+    # make libguestfs use /dev/shm as tmpdir
+    export LIBGUESTFS_CACHEDIR="/dev/shm"
```

Comment:
1. I was surprised it did not add as much as I hoped when I tried it manually previously - in Lago itself (in the sysprep stage). But why not change it there as well (the same way we control the backend).
2. I have a suspicion other parameters should also go there:
LIBGUESTFS_TMPDIR
XDG_RUNTIME_DIR
TMPDIR